src/components/confirm/__spec__.js

```diff
@@ -12,7 +12,7 @@ describe('Confirm', () => {
 
     wrapper = shallow(
       <Confirm
-        open
+        open={ false }
```

Comment:
again why has this changed?

Reply:
Sorry We will change this back. We were fiddling around and it got left in place.

Reply:
Actually we've changed  these back to having the Confirm open.